### DISCUSSION TITLE
Is this still maintained?

### DISCUSSION THREAD

**Discussion by N0m4n904** [opencardev/crankshaft#657] [General]
Hi everyone, 
I am using crankshaft now for half a year and I was wondering if the devs are still working on it.
Does anyone know if they said something about this? 
It's sad that such a great peace of software has not received a update for over a year now.

**Comment by Jingo-Strange** [comment]
That's the best part about open source software, even if the original devs have moved on, nothing stops someone else from picking up the torch.

**Comment by matt2005** [comment]
Hi,
I intend to update it soon. I've had a lot of personal changes going on so this has taken a back seat. 
I've also had to change cars a few times and haven't been able to get it reinstalled.

**Comment by mi-hol** [comment]
@N0m4n904 see https://github.com/opencardev/crankshaft/discussions/693
